app/controllers/agentAppointment/AppointAgentController.scala

```diff
@@ -100,13 +100,17 @@ class AppointAgentController @Inject()(
                                                       ): Action[AnyContent] = authenticated.async { implicit request =>
     for {
       agentOrganisation <- accounts.withAgentCode(agentCode.toString)
-      response <- agentRelationshipService.getMyOrganisationPropertyLinksWithAgentFiltering(params, AgentPropertiesParameters(
-        agentCode = agentCode,
-        checkPermission = AgentPermission.fromName(checkPermission).getOrElse(StartAndContinue),
-        challengePermission = AgentPermission.fromName(challengePermission).getOrElse(StartAndContinue),
-        pageNumber = pagination.page,
-        pageSize = pagination.pageSize),
-        request.organisationAccount.id, agentOrganisation.fold(throw new IllegalArgumentException("agent organisation required."))(_.id))
+      response <- agentRelationshipService.getMyOrganisationPropertyLinksWithAgentFiltering(
+        params = params,
+        pagination = AgentPropertiesParameters(
+          agentCode = agentCode,
+          checkPermission = AgentPermission.fromName(checkPermission).getOrElse(StartAndContinue),
+          challengePermission = AgentPermission.fromName(challengePermission).getOrElse(StartAndContinue),
+          pageNumber = pagination.page,
+          pageSize = pagination.pageSize,
+          agentAppointed = agentAppointed.getOrElse("BOTH")
```

Comment:
what is "BOTH"?

Reply:
we default to BOTH because that will bring back all of the user's properties

Reply:
BOTH is the default on the API as well

Reply:
then `AgentPropertiesFilter.Both.name` ?

Reply:
updated